### ISSUE TITLE
Ci failes. 

### ISSUE THREAD

**Issue by florianlenz** [Bit-Nation/BITNATION-Pangea-mobile#84]
## Issue
Sometimes bitrise fails. It seem's to be a kind of random. The error: 
```
▸ Copying /Users/vagrant/git/node_modules/react-native-vector-icons/Fonts/MaterialCommunityIcons.ttf
▸ Compiling LaunchScreen.xib
▸ Processing Preprocessed-Info.plist
▸ Generating 'Pangea.app.dSYM'
▸ Running script 'Bundle React Native code and images'
the transform cache was reset.
❌  error: File /Users/vagrant/Library/Developer/Xcode/DerivedData/Pangea-aiqbjxbwbwzexecwjgzgojuzeleo/Build/Intermediates.noindex/ArchiveIntermediates/Pangea/BuildProductsPath/Release-iphoneos/Pangea.app/main.jsbundle does not exist. This must be a bug with
** ARCHIVE FAILED **
The following build commands failed:
PhaseScriptExecution Bundle\ React\ Native\ code\ and\ images /Users/vagrant/Library/Developer/Xcode/DerivedData/Pangea-aiqbjxbwbwzexecwjgzgojuzeleo/Build/Intermediates.noindex/ArchiveIntermediates/Pangea/IntermediateBuildFilesPath/Pangea.build/Release-iphoneos/Pangea.build/Script-00DD1BFF1BD5951E006B06BC.sh
(1 failure)
[22:43:23]: Exit status: 65
```

have a look at [this](https://github.com/facebook/react-native/issues/15432)

## Acceptance criteria
- [ ] Builds should pass

**Comment by florianlenz** [comment]
I opened an issue at the promise lib: https://github.com/stefanpenner/es6-promise/issues/316 but I am not sure if it's really a bug on there side since it's working locally on my machine but not in the docker container.

**Comment by seland** [comment]
Closed by #105.
